.github/workflows/rust.yml

```diff
@@ -91,7 +91,7 @@ jobs:
           cargo nextest run --profile ci
       - name: benchmark (smoke)
         run: |
-          cargo run --package sui-benchmark --bin stress -- --log-path /tmp/stress.log --num-client-threads 10 --num-server-threads 24 --num-transfer-accounts 2 bench --target-qps 100 --num-workers 10  --transfer-object 50 --shared-counter 50 --run-duration 10s --stress-stat-collection
+          cargo run --package sui-benchmark --bin stress -- --log-path /tmp/stress.log --num-client-threads 10 --num-server-threads 24 --num-transfer-accounts 2 bench --target-qps 100 --num-workers 10  -w transfer_object=50 -w shared_counter=50 --run-duration 10s --stress-stat-collection
           pushd narwhal/benchmark && fab smoke && popd
```

Comment:
@phoenix-o since this is a breaking change, it will require us to update all environments with updated cli params in sui-operations and then if people are running an older commit, it will require careful co-ordination. I am wondering if there is any way to avoid that?

Reply:
yes, I can make it temporary backward compatible if necessary
leave some of the original parameters around and just proxy them internally into the weights map

Reply:
cool, that's a great idea. Thanks!